StartioAdmobMediation/StartioAdmobBannerAdapter.m

```diff
@@ -0,0 +1,147 @@
+/**
+ * Copyright 2022 Start.io Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import "StartioAdmobBannerAdapter.h"
+#import "StartioAdmobAdapterConfiguration.h"
+#import "StartioAdmobParameters.h"
+#import "StartioAdmobBannerAd.h"
+#import "STAAdPreferences+AdMob.h"
+
+@import GoogleMobileAds;
+
+@interface StartioAdmobBannerAdapter () <STABannerDelegateProtocol>
+
+@property (nonatomic, strong) UIView* fakeView;
+@property (nonatomic, strong) STABannerView* bannerView;
+@property (nonatomic, assign) BOOL isAutoSizeBanner;
+@property (nonatomic, copy) GADMediationBannerLoadCompletionHandler completionHandler;
+@property (nonatomic, weak) id<GADMediationBannerAdEventDelegate> delegate;
```

Comment:
here and in other adapters in AdMob sample such delegates are not weak, but retain cycle is broken by setting original completion handler to nil. are you sure your approach is ok?